### DISCUSSION TITLE
Notification template with Application annotation

### DISCUSSION THREAD

**Discussion by duncaan** [argoproj/argo-cd#13010] [Q&A]
Hello 👋 
I'm attempting to use an annotation that I set in the Argo Application inside of a notification template, something like this:
```
        slack:
          attachments: |
            [{
              {
                "title": "Image Tag",
                "value": "{{ .app.metadata.annotations["example.com/imageTag"] }}",
                "short": true
              }
```
This notation seems to work in the triggers section, but not in the template.  Is there a way that I can use this?
Thanks in advance!

**Comment by duncaan** [comment]
I'm closing this - I just changed the annotation to not include a `.` or `/` and everything was fine.

**Reply by dbaudracco** [reply]
For future reference, you can use the gotmpl function `index` to retrieve the right annotation
```
                "value": "{{ index .app.metadata.annotations "example.com/imageTag" }}",
```

**Reply by Den4200** [reply]
Hi, this does not seem to work in my testing. The expression is simply not substituted at all.

I still can't find a solution to this other than renaming the annotations to not contain `.`s. Any help would be appreciated.
